.changeset/bright-kings-relate.md

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
Happy to update this if the format is off! The [reviewing guide ](https://github.com/backstage/backstage/blob/master/REVIEWING.md)cites combining changesets for _unrelated_ information as bad form, but I thought as these changes are intertwined it made sense to connect them for easier traceability around when the move happens.

Omitted the example-app changeset since it's private, and omitted core-compat-api & create-app as my interpretation was they were internal changes, but less confident about those two/can imagine an argument that they should be changesets as well.

Reply:
You'll for sure need a changeset for `create-app` 👍

Reply:
> _Omitted the example-app changeset since it's private, and omitted core-compat-api_

Yep!  That makes sense to me.

> _as these changes are intertwined it made sense to connect them for easier traceability around when the move happens._

I think it definitely makes sense for `create-app` and `plugin-catalog` to include these details.  If there's any opportunity to reduce redundant info, it might be on `plugin-catalog-react`, where you could maybe get away with a one-liner like...

```md
---
'@backstage/plugin-catalog-react': minor
---

This package is now the source of truth for the `<EntitySwitch>` component, as well as the related utilities `isComponentType()`, `isEntityWith()`, `isKind()`, `isNamespace()`, and `isResourcetype()`.
```